x-pack/legacy/plugins/reporting/export_types/png/server/execute_job/index.ts

```diff
@@ -20,8 +25,15 @@ import { generatePngObservableFactory } from '../lib/generate_png';
 
 export const executeJobFactory: ExecuteJobFactory<ESQueueWorkerExecuteFn<
   JobDocPayloadPNG
->> = function executeJobFactoryFn(server: ServerFacade) {
-  const generatePngObservable = generatePngObservableFactory(server);
+>> = function executeJobFactoryFn(
+  server: ServerFacade,
+  { browserDriverFactory }: { browserDriverFactory?: HeadlessChromiumDriverFactory } = {}
+) {
+  if (!browserDriverFactory) {
```

Comment:
Note: `browserDriverFactory` is an optional field in the execute job opts. See `x-pack/legacy/plugins/reporting/server/routes/generate_from_savedobject_immediate.ts|49`